src/commands/env/logdrain/remove.ts

```diff
@@ -67,37 +66,31 @@ export default class LogDrainRemove extends Command {
     }
 
     if (flags.environment) {
-      cli.warn(messages.getMessage('flags.environment.deprecation'));
+      this.warn(messages.getMessage('flags.environment.deprecation'));
     }
 
     if (flags.url) {
-      cli.warn(messages.getMessage('flags.url.deprecation'));
+      this.warn(messages.getMessage('flags.url.deprecation'));
     }
 
     const appName = await resolveAppNameForEnvironment(targetCompute);
     try {
+      cli.action.start(`Deleting drain for environment ${herokuColor.app(targetCompute)}`);
+
       await this.client.delete<Heroku.LogDrain>(`/apps/${appName}/log-drains/${encodeURIComponent(url)}`);
-      if (flags.json) {
-        cli.styledJSON({
-          status: 0,
-          result: null,
-          warnings: [],
-        });
-        return;
-      } else {
-        cli.action.start(`Deleting drain for environment ${herokuColor.app(targetCompute)}`);
-
-        cli.action.stop();
-      }
+
+      cli.action.stop();
+
+      return [];
```

Comment:
I won't hold up this PR for this but you should return _something_.
For these types of commands we often return the thing/object that was removed with a success message.

Reply:
10-4